management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument("--poll_seconds", type=float, default=3)
+        parser.add_argument("--max_retries", type=int, default=60)
+
+    def handle(self, *args, **options):
+        max_retries = options["max_retries"]
+        poll_seconds = options["poll_seconds"]
+
+        for retry in range(max_retries):
+            try:
+                connection.ensure_connection()
+            except OperationalError as ex:
+                self.stdout.write(
```

Comment:
Consider using `self.stdout.write(self.style.WARNING(...))` to make the output more visually distinct when the database is unavailable. This can help in quickly identifying issues in logs.